plugins/microsoft_log_analytics/plugin.spec.yaml

```
@@ -0,0 +1,131 @@
+plugin_spec_version: v2
+extension: plugin
+products: [insightconnect]
+name: microsoft_log_analytics
+title: Microsoft Log Analytics
+description: Edit, run log queries with data in Azure Monitor Logs
+version: 1.0.0
+vendor: rapid7
+support: rapid7
+supported_versions: ["2022-03-18"]
+status: []
+resources:
+  source_url: https://github.com/rapid7/insightconnect-plugins/tree/master/plugins/microsoft_log_analytics
+  license_url: https://github.com/rapid7/insightconnect-plugins/blob/master/LICENSE
+  vendor_url: https://www.microsoft.com/
+tags:
+  - cloud
+  - microsoft
+  - log analytics
+  - azure
+hub_tags:
+  use_cases: [data_utility]
+  keywords: [microsoft, azure]
+  features: []
+enable_cache: false
+connection:
+  client_id:
+    title: Client ID
+    description: The application ID that the application registration portal assigned
+      to your app
+    type: string
+    required: true
+    example: 5cdad72f-c848-4df0-8aaa-ffe033e75d57
+  client_secret:
+    title: Client Secret
+    description: The application secret that you generated for your app in the app
+      registration portal
+    type: credential_secret_key
+    required: true
+    example: 5cdad72f-c848-4df0-8aaa-ffe033e75d57
+  subscription_id:
+    title: Subscription ID
+    description: Current subscription identifier that Azure application portal assigned to
+    type: string
+    required: true
+    example: 5cdad72f-c848-4df0-8aaa-ffe033e75d57
+  resource_group_name:
+    title: Resource Group Name
+    description: Name of the resource group
+    type: string
+    required: true
+    example: ExampleResourceGroupName
+  tenant_id:
+    title: Tenant ID
+    description: This is active directory ID
```

```suggestion
    description: This is an Active Directory ID
```
I think proper names are spelled with capital letters?